test/object-store/notifications-fuzzer/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 macro(build_fuzzer_variant variant)
     add_executable(${variant} command_file.hpp command_file.cpp ${variant}.cpp)
-    target_link_libraries(${variant} realm-object-store)
+    target_link_libraries(${variant} ObjectStore)
```

Comment:
By specifying the library name rather than the cmake target this was bypassing flag propagation. I didn't think that was intentional, so I change it to use the target name.